src/plugins/credential_management/public/components/utils.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { SavedObjectsClientContract } from 'src/core/public';
+import { CredentialsTableItem } from './types';
+
+export async function getCredentials(savedObjectsClient: SavedObjectsClientContract) {
+  return await (savedObjectsClient
+    .find({
+      type: 'credential',
+      fields: ['id', 'title', 'description', 'credentialMaterials'],
+      perPage: 10000,
+    })
+    .then((response) =>
+      response.savedObjects
+        .map((source) => {
+          const id = source.id;
+          const title = source.get('title');
+          const description = source.get('description');
+          const credentialMaterialsType = source.get('credentialMaterials')
+            ?.credentialMaterialsType;
+          return {
+            id,
+            title,
+            description,
+            credentialMaterialsType,
+            sort: `${title}`,
+          };
+        })
+        .sort((a, b) => {
+          if (a.sort < b.sort) {
+            return -1;
+          }
+
+          if (a.sort > b.sort) {
+            return 1;
+          }
+
+          return 0;
+        })
```

Comment:
We should use `localeCompare` here or even better use `Intl.Collator`.

Reply:
This module has been delete &  for data source management .. I have removed sort method as table is taking care of sort